.github/workflows/tests.yaml

```diff
@@ -29,6 +29,8 @@ jobs:
       uses: actions/checkout@v2
     - name: run all tests
       run: make test.all
+      env:
+        KONG_ENTERPRISE_LICENSE: ${{ secrets.KONG_ENTERPRISE_LICENSE }}
```

Comment:
This secret wont be accessible unless you attach an environment to the job, e.g.:

```
jobs:
  tests-and-coverage:
    environment: "ENTERPRISE"
```

Reply:
Ok looks like the `environment` is added to the job now :+1: